text.cabal

```diff
@@ -255,6 +256,7 @@ test-suite tests
     deepseq,
     directory,
     ghc-prim,
+    mtl,
```

Comment:
Is it possible to avoid dependency on `mtl`?

Reply:
Yes